### DISCUSSION TITLE
Grouping of Terraform provider updates did not work

### DISCUSSION THREAD

**Discussion by ahabb1** [renovatebot/renovate#13815] [Request Help] [answered]
I'm trying to group updates for Yandex Terraform provider with no luck. I have this in two separate folders and i need Renovate to update in both places in the same PR.
The update:
<html>
<body>

Package | Type | Update | Change
-- | -- | -- | --
yandex (source) | required_provider | minor | 0.69.0 -> 0.70.0

</body>
</html>

Renovate configuration :
```
{
  ...
  "extends": [
    "config:base",
    ":semanticCommits",
    ":semanticCommitTypeAll(chore)",
    ":semanticCommitScopeDisabled"
  ],
  "additionalBranchPrefix": "{{baseDir}}-",
  "commitMessageTopic": "{{depName}}",
  "commitMessageExtra": "",
  "commitMessageSuffix": "",
  "packageRules": [
    ...
    {
      "matchPackagePatterns": [".*yandex.*"],
      "groupName": "yandex"
    },
  ]
}
```
Am i doing something wrong or does having the directory name in the branch name ruin the grouping?

**Comment by rarkins** [comment, marked as answer]
Your problem is likely additionalBranchPrefix

**Reply by ahabb1** [reply]
Is it possible to override that option? As to add a different additionalBranchPrefix for this package rule?

**Reply by rarkins** [reply]
Of course. But you should maybe question whether you should be adding the field using a selective package rule in the first place

**Reply by ahabb1** [reply]
I have just tested and it was indeed the _additionalBranchPrefix_ that was causing the behavior above.
